examples/CMakeLists.txt

```
@@ -1675,3 +1675,35 @@ add_impactx_test(solenoid-softedge-solvable.py
     examples/solenoid_softedge/analysis_solenoid_softedge_solvable.py
     OFF  # no plot script yet
 )
+
+# Expanding unbunched beam in free space with 2D space charge ######################
+#
+# with space charge
+add_impactx_test(expanding-fft-2d
+    examples/expanding_beam/input_expanding_fft_2D.in
+    ON   # ImpactX MPI-parallel
+    examples/expanding_beam/analysis_expanding_fft_2D.py
+    OFF  # no plot script yet
+)
+add_impactx_test(expanding-fft-2d.py
+    examples/expanding_beam/run_expanding_fft_2D.py
+    OFF   # ImpactX MPI-parallel
+    examples/expanding_beam/analysis_expanding_fft_2D.py
+    OFF  # no plot script yet
+)
+
+# FODO cell with 2D space charge using particle tracking ######################
+#
+# with space charge
+add_impactx_test(fodo-2d-sc
+    examples/fodo_space_charge/input_fodo_2d_sc.in
+    ON   # ImpactX MPI-parallel
+    examples/fodo_space_charge/analysis_fodo_2d_sc.py
+    OFF  # no plot script yet
+)
+add_impactx_test(fodo-2d-sc.py
+    examples/fodo_space_charge/run_fodo_2d_sc.py
+    OFF   # ImpactX MPI-parallel
```

If we want 100% the same AMReX blocks ("grids output"):
```suggestion
    ON    # ImpactX MPI-parallel
```